src/app/pages/system/advanced/cron/cron-list/cron-list.component.html

```diff
@@ -0,0 +1,51 @@
+<ng-template #pageHeader>
+  <ix-page-title-header>
+    <ix-search-input (search)="onListFiltered($event)"></ix-search-input>
+
+    <ix-table-columns-selector [columns]="columns" (changed)="columnsChanged()"></ix-table-columns-selector>
```

Comment:
If you rename `(changed)` to `(columnsChange)` we may be able to use [two way binding](https://angular.io/guide/two-way-binding) here.